frontends/infinite-corridor/src/scss/fieldpage.scss

```diff
@@ -31,8 +31,8 @@ $avatarSize: 60px;
 $carouselSpacing: 24px;
 .ic-carousel-card {
   height:100%;
-  margin-left: $carouselSpacing/2;
-  margin-right: $carouselSpacing/2;
+  margin-left: $carouselSpacing*0.5;
+  margin-right: $carouselSpacing*0.5;
```

Comment:
This is from running `npx sass-migrator division **/*.scss`, which fixed some deprecated Sass syntax.

(*At one point on this branch I had a failing build that I thought might be related to a sass issue, so I was trying to clean up some warnings.*)